package/Editor/Isometric Workflow/IsometricWorkflow.cs

```diff
@@ -0,0 +1,300 @@
+        {
+            if (isVisible)
+                return;
+
+            PromptFetcher.IsWorkflow = true;
+
+            GetWindow<IsometricWorkflow>();
+
+            settings = IsometricWorkflowSettings.GetSerializedSettings();
+        }
+
+        public void Restart()
+        {
+            PromptFetcher.IsWorkflow = true;
+
+            var isometricWorkflow = (IsometricWorkflow)GetWindow(typeof(IsometricWorkflow));
+            
+            isometricWorkflowUI.Init(isometricWorkflow);
+            isometricWorkflow.currentStep = Step.Base;
+
+            settings = IsometricWorkflowSettings.GetSerializedSettings();
+        }
+
+        /// <summary>
+        /// Auto add some asset name as an example
+        /// </summary>
+        /// <returns></returns>
+        public void FillAssetSamples()
+        {
+            List<string> samples = new()
+            {
+                "Tavern",
+                "Hospital",
+                "Police Station",
+                "Rocket Launcher",
+                "Factory",
+                "Treehouse",
+                "Arena",
+                "Temple",
+                "Church",
+                "Building Block"
+            };
+            assetList.AddRange(samples);//
+        }
+
+        private void CreateGUI()
+        {
+            var isometricWorkflow = (IsometricWorkflow)GetWindow(typeof(IsometricWorkflow));
+            isometricWorkflowUI.Init(isometricWorkflow);
+        }
+
+        /// <summary>
+        /// Call the prompt window to generate one image per asset in the assetlist
+        /// </summary>
+        /// <param name="_onRequestSent">callback when ALL requests has been sent</param>
+        public void GenerateImages(Action _onRequestSent)
+        {
+            Texture2D baseTexture = null;
+            switch (selectedBase)
+            {
+                case Base.None:
+                    baseTexture = null;
+                    break;
+                case Base.Square:
+                    baseTexture = settings.squareBaseTexture;
+                    break;
+                case Base.Custom:
+                    baseTexture = isometricWorkflowUI.customTexture;
+                    break;
+                default:
+                    break;
+            }
+
+            inferenceIdByAssetList.Clear();
+
+            int totalRequests = assetList.Count;
+            int completedRequests = 0;
+
+            foreach (string assetName in assetList)
+            {
+                string tempName = assetName;
+                bool useBaseTexture = baseTexture != null;
+                string modelName = settings.isometricModels.Find(x => x.style == selectedModel).modelData.name;
+                PromptWindow.GenerateImage(modelName, useBaseTexture, useBaseTexture, useBaseTexture ? baseTexture : null, 4, $"isometric, solo, centered, solid color background, {selectedTheme}, {assetName}", 30, 1024, 1024, 6, "-1", useBaseTexture, 0.8f,
```

Comment:
I think we should make the prompt value "isometric, solo, centered, solid color background" as a variable so we can change it easily

Reply:
Now it's fix in a variable and that's mean in the futur we can have a custom prompt on it ?